virtcontainers/cgroups_test.go

```diff
@@ -197,3 +197,74 @@ func TestUpdateCgroups(t *testing.T) {
 	err = s.cgroupsDelete()
 	assert.NoError(err)
 }
+
+func TestIsSystemdCgroup(t *testing.T) {
+	assert := assert.New(t)
+
+	assert.False(isSystemdCgroup("/kata/afhts2e5d4g5s"))
+	assert.True(isSystemdCgroup("slice:kata:afhts2e5d4g5s"))
+}
+
+func TestValidCgroupPath(t *testing.T) {
+	assert := assert.New(t)
+	var err error
+	var path string
+
+	renamedCgroupPath, err := renameCgroupPath(defaultCgroupPath)
+	assert.NoError(err)
+
+	path, err = validCgroupPath("../../../", false)
+	assert.NoError(err)
+	assert.Equal(renamedCgroupPath, path)
+
+	path, err = validCgroupPath("../", false)
```

Comment:
Nit: this would be clearer more compact if you used a table-based approach (https://github.com/kata-containers/tests/blob/master/Unit-Test-Advice.md#table-driven-tests).